<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScaleInterface</class>
 <widget class="QWidget" name="ScaleInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>音阶查看</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>531</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="BodyLabel" name="scaleSelectionLabel">
      <property name="text">
       <string>音阶选择</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="ComboBox" name="CComboBox"/>
    </item>
    <item>
     <widget class="ComboBox" name="bComboBox"/>
    </item>
    <item>
     <widget class="ComboBox" name="keyComboBox"/>
    </item>
    <item>
     <widget class="SwitchButton" name="SwitchButton">
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="onText">
       <string/>
      </property>
      <property name="offText">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="ScrollArea" name="ScrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>905</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border: none;</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>905</width>
      <height>1800</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>1800</height>
     </size>
    </property>
    <widget class="CardWidget" name="CardWidget">
     <property name="geometry">
      <rect>
       <x>-5</x>
       <y>0</y>
       <width>910</width>
       <height>1800</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScrollArea</class>
   <extends>QScrollArea</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
